reportportal/templates/api-deployment.yaml

```diff
@@ -53,6 +53,10 @@ spec:
         {{- else }}
           value: "{{ .Values.rabbitmq.endpoint.password }}"
         {{- end }}
+        - name: RP_AMQP_ADDRESSES
+          value: "{{ .Values.rabbitmq.endpoint.protocol }}://{{ .Values.rabbitmq.endpoint.user }}:{{ .Values.rabbitmq.endpoint.password }}@{{ .Values.rabbitmq.endpoint.address | default (printf "%s-rabbitmq.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}:{{ .Values.rabbitmq.endpoint.port }}"
+        - name: RP_AMQP_API_ADDRESS
+          value: "{{ .Values.rabbitmq.endpoint.apiprotocol }}://{{ .Values.rabbitmq.endpoint.user }}:{{ .Values.rabbitmq.endpoint.password }}@{{ .Values.rabbitmq.endpoint.address | default (printf "%s-rabbitmq.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}/api"
```

Comment:
Due to missed port here I'm getting such error on service-api deployment:
```none
With this value I'm getting this error in `serviceapi` after deployment:
I/O error on PUT request for "http://rabbit-rabbitmq.default.svc.cluster.local/api/vhosts/analyzer": Connect to rabbit-rabbitmq.default.svc.cluster.local:80 [rabbit-rabbitmq.default.svc.cluster.local/10.112.8.16] failed: Operation timed out
```
And there is also issue with how you generate default value.